### PR TITLE
DAT-2593: Added max height to modals

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
https://wikia-inc.atlassian.net/browse/CE-1492

### CODE REVIEW

File: resources/wikia/libraries/jquery/modal/jquery.wikia.modal.js [hunk 1]
```diff
@@ -80,10 +92,12 @@ $.fn.extend({
 			// find tabs with .modal-tabs class and move them outside modal content
 			var modalTabs = wrapper.find('.modal-tabs');
 			if (modalTabs.exists()) {
-				modalTabs.insertBefore(wrapper.find('.modalContent'));
+				modalTabs.insertBefore($modalContent);
 			}
 		}
 
+		$modalContent.css('max-height', this.getMaxModalContentHeight(settings.topOffset, headline, modalTabs));
```
Comment (lukaszkonieczny) [inline resources/wikia/libraries/jquery/modal/jquery.wikia.modal.js]: ```headline``` and ```modalTabs``` can be ```undefined```

**Comment by lukaszkonieczny** [comment]
LGTM

File: resources/wikia/libraries/jquery/modal/jquery.wikia.modal.js [hunk 2]
```diff
@@ -62,7 +74,7 @@ $.fn.extend({
 
 		if(!settings.noHeadline) {
 			//set up headline
-			var headline = wrapper.find("h1:first");
+			headline = wrapper.find("h1:first");
```
Comment (bkoval) [inline resources/wikia/libraries/jquery/modal/jquery.wikia.modal.js]: We could use single quotes here :)

**Comment by SebastianMarzjan** [comment]
@lukaszkonieczny @lgarczewski  
is that still valid?

**Comment by SebastianMarzjan** [comment]
@lukaszkonieczny @lgarczewski 

**POKE**

**Comment by dianafa** [comment]
can we close this PR? @lukaszkonieczny

**Pull request closed without merging**